src/pages/docs/chat/connect.mdx

```diff
@@ -209,3 +209,72 @@ subscription.unsubscribe()
 <If lang="javascript,swift,kotlin">
 The discontinuity handler is accessible via the <If lang="javascript">[Room](https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Room.html#onDiscontinuity)</If><If lang="swift">[Room](https://sdk.ably.com/builds/ably/ably-chat-swift/main/AblyChat/documentation/ablychat/room)</If><If lang="kotlin">[Room](https://sdk.ably.com/builds/ably/ably-chat-kotlin/main/dokka/chat-android/com.ably.chat/-room/index.html)</If> object.
 </If>
+
+## Logging <a id="logging"/>
```

Comment:
I think this needs to be removed as we now duplicate it here and in the setup page.